src/sentry/lang/native/symbolicator.py

```diff
@@ -420,3 +428,32 @@ def query_task(self, task_id):
 
     def reset_worker_id(self):
         self.worker_id = uuid.uuid4().hex
+
+
+class _Backoff:
+    """
+    Creates a new exponential backoff.
+    """
+
+    def __init__(self, initial, max):
+        """
+        :param initial: The initial backoff time in seconds.
+        :param max: The maximum backoff time in seconds.
+        """
+        self.initial = initial
+        self.max = max
```

Comment:
nit: Any reason to make these "public"?

```suggestion
        self._initial = initial
        self._max = max
```

Reply:
Yeah, because there is nothing wrong when the caller wants to change them, I just made the `_current` 'private' because you shouldn't mess with that and is internal state.